packages/components/atoms/f-form-field/src/components/FormLabel.vue

```diff
@@ -9,7 +9,9 @@
                 [$style['c-formField-label--inline']]: isInline,
                 [$style['c-formField-label--disabled']]: isDisabled
             }
-        ]">
+        ]"
+        @mouseenter="toggleHoverState"
+        @mouseleave="toggleHoverState">
```

Comment:
Can delete this – must have missed this when deleting the hover class stuff.

Reply:
think it was me 🤦